dlrover/examples/torch_nanogpt_debug_job.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: elastic.iml.github.io/v1alpha1
+kind: ElasticJob
+metadata:
+  name: torch-nanogpt-debug
+  namespace: dlrover
+spec:
+  distributionStrategy: AllreduceStrategy
+  optimizeMode: single-job
+  replicaSpecs:
+    worker:
+      replicas: 2
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: main
+              # yamllint disable-line rule:line-length
+              image: easydl/dlrover-train-nanogpt:test
+              imagePullPolicy: Never
```

Comment:
IfNotPresent

Reply:
Tks for reviewing. However, this is not the final version for this yaml. Maybe 'IfNotPresent' is better for test example yaml not this debug yaml.